_rules/css-orientation-lock-media-queries-b33eff.md

```diff
@@ -0,0 +1,209 @@
+---
+id: b33eff
+name: Page has no orientation lock specified using CSS media queries
+rule_type: atomic
+description: |
+  This rule checks that page content is not locked to any single display orientation using CSS media queries.
+accessibility_requirements:
+ wcag20:1.3.4: # Orientation
+  forConformance: true
+  failed: not satisfied
+  passed: further testing needed
+  inapplicable: further testing needed
+input_aspects:
+- DOM Tree
+- CSS Styling
+authors:
+- Jey Nandakumar
+- Audrey Maniez
+---
+
+## Applicability
+
+The rule applies to any element that is [visible](#visible) and has a CSS [transform](https://developer.mozilla.org/en-US/docs/Web/CSS/transform) property with any of the below [transform-functions](https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function)
+
+- [rotate transform-function](https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function/rotate)
+- [rotate3d transform-function](https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function/rotate3d)
+- [rotateX transform-function](https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function/rotateX)
+- [rotateY transform-function](https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function/rotateY)
+- [rotateZ transform-function](https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function/rotateZ)
+- [matrix transform-function](https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function/matrix)
+- [matrix3d transform-function](https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function/matrix3d)
```

Comment:
I'd suggest listing these as:

- [`rotate()`](https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function/rotate)
- [`rotate3d()`](https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function/rotate3d)
- ...

Reply:
Maybe also add a note explaining why these functions are relevant? Something along the lines of:

> These specific transformation functions are of interest to this rule as they have the potential to affect the rotation of a given element.

Reply:
I do not think the **note** is important, the fact that we are listing these functions and describing what we need in the expectation is plenty in my opinion.

Also the word `rotation` in your above suggestion will make it ambiguous.

Reply:
> the fact that we are listing these functions and describing what we need in the expectation is plenty in my opinion.

For people who are intimately familiar with the CSS transforms specification, sure. For people who are not there's a missing link though. As for making the term "rotation" unambiguous, I'd just reference https://drafts.csswg.org/css-transforms-2/#Rotate3dDefined.